Android build fails with error: jni/src/gui/touchscreengui.h:30:10: fatal error: 'game.h' file not found

##### Issue type
<!-- Pick one below and delete others -->
 - Build issue
 
##### Minetest version
<!--
Paste Minetest version between quotes below
If you are on a devel version, please add git commit hash
You can use `minetest --version` to find it.
-->
5.0.0 (Commit: a7025c72730adcd41648613cdad93f4e6507c3b4)

##### OS / Hardware
<!-- General information about your hardware and operating system -->
Operating system: Ubuntu Linux 18.04.1 LTS
CPU:  Pentium(R) Dual-Core  CPU      E6300  @ 2.80GHz

##### Summary
<!-- Describe your problem here -->
Trying to build for android from commit a7025c72730adcd41648613cdad93f4e6507c3b4 fails with error: 
```
In file included from jni/src/client/inputhandler.h:29:
jni/src/gui/touchscreengui.h:30:10: fatal error: 'game.h' file not found
```
**Build environment:** Ubuntu 18.04.1 LTS with all required android dependencies installed, running on Virtualbox 5.2.22
local.properties contents:
```
ndk.dir = /home/juozas/android-ndk-r17c
sdk.dir = /home/juozas/android-sdk-linux
```

<details>
<summary>Full log</summary>

```
juozas@ubuntu1:~/minetest/build/android$ make debug
]export NDEBUG=;                                                            \
export BUILD_TYPE=debug;                                                   \
make apk
make[1]: Entering directory '/home/juozas/minetest/build/android'
nothing to be done for freetype
nothing to be done for irrlicht
changed /home/juozas/minetest/games/minetest_game/.git/ORIG_HEAD
assets changed, refreshing...
make[2]: Entering directory '/home/juozas/minetest/build/android'
./gradlew cleanAssets

BUILD SUCCESSFUL in 2s
1 actionable task: 1 up-to-date
make[2]: Leaving directory '/home/juozas/minetest/build/android'
find: ‘./Minetest/games/minetest_game/.git’: Toks failas ar aplankas neegzistuoja
nothing to be done for iconv
nothing to be done for gmp
nothing to be done for openssl
nothing to be done for curl
nothing to be done for openal
nothing to be done for libogg/libvorbis
android_version_githash.h changed, updating...
Android NDK: APP_PLATFORM not set. Defaulting to minimum supported version android-14.    
make[2]: Entering directory `/home/juozas/minetest/build/android'
[armeabi-v7a] Compile++ thumb: minetest <= ban.cpp
[armeabi-v7a] Compile++ thumb: minetest <= camera.cpp
In file included from jni/../jni/src/client/camera.cpp:20:
In file included from jni/../jni/src/client/camera.h:22:
In file included from jni/src/irrlichttypes_extrabloated.h:28:
In file included from deps/irrlicht/include/IMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= cavegen.cpp
In file included from jni/../jni/src/mapgen/cavegen.cpp:29:
In file included from jni/../jni/src/mapgen/mg_biome.h:23:
In file included from jni/src/objdef.h:23:
In file included from jni/src/porting.h:36:
In file included from deps/irrlicht/include/irrlicht.h:60:
In file included from deps/irrlicht/include/IAnimatedMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= chat.cpp
[armeabi-v7a] Compile++ thumb: minetest <= client.cpp
In file included from jni/../jni/src/client/client.cpp:25:
In file included from jni/../jni/src/client/client.h:22:
In file included from jni/../jni/src/client/clientenvironment.h:23:
In file included from deps/irrlicht/include/ISceneManager.h:21:
In file included from deps/irrlicht/include/ISkinnedMesh.h:9:
In file included from deps/irrlicht/include/IBoneSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= clientenvironment.cpp
In file included from jni/../jni/src/client/clientenvironment.cpp:22:
In file included from jni/../jni/src/client/client.h:22:
In file included from jni/../jni/src/client/clientenvironment.h:23:
In file included from deps/irrlicht/include/ISceneManager.h:21:
In file included from deps/irrlicht/include/ISkinnedMesh.h:9:
In file included from deps/irrlicht/include/IBoneSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= clientiface.cpp
In file included from jni/../jni/src/clientiface.cpp:21:
In file included from jni/src/clientiface.h:28:
In file included from jni/src/porting.h:36:
In file included from deps/irrlicht/include/irrlicht.h:60:
In file included from deps/irrlicht/include/IAnimatedMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= clientmap.cpp
In file included from jni/../jni/src/client/clientmap.cpp:20:
In file included from jni/../jni/src/client/clientmap.h:22:
In file included from jni/src/irrlichttypes_extrabloated.h:28:
In file included from deps/irrlicht/include/IMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= clientmedia.cpp
In file included from jni/../jni/src/client/clientmedia.cpp:22:
In file included from jni/../jni/src/client/client.h:22:
In file included from jni/../jni/src/client/clientenvironment.h:23:
In file included from deps/irrlicht/include/ISceneManager.h:21:
In file included from deps/irrlicht/include/ISkinnedMesh.h:9:
In file included from deps/irrlicht/include/IBoneSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= clientobject.cpp
In file included from jni/../jni/src/client/clientobject.cpp:20:
In file included from jni/../jni/src/client/clientobject.h:22:
In file included from jni/src/irrlichttypes_extrabloated.h:28:
In file included from deps/irrlicht/include/IMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= clouds.cpp
In file included from jni/../jni/src/client/clouds.cpp:20:
In file included from jni/src/client/renderingengine.h:26:
In file included from jni/src/irrlichttypes_extrabloated.h:28:
In file included from deps/irrlicht/include/IMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= collision.cpp
In file included from jni/../jni/src/collision.cpp:27:
In file included from jni/src/client/clientenvironment.h:23:
In file included from deps/irrlicht/include/ISceneManager.h:21:
In file included from deps/irrlicht/include/ISkinnedMesh.h:9:
In file included from deps/irrlicht/include/IBoneSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= content_cao.cpp
In file included from jni/../jni/src/client/content_cao.cpp:20:
In file included from deps/irrlicht/include/ICameraSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= content_cso.cpp
In file included from jni/../jni/src/client/content_cso.cpp:20:
In file included from jni/../jni/src/client/content_cso.h:22:
In file included from jni/src/irrlichttypes_extrabloated.h:28:
In file included from deps/irrlicht/include/IMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= content_mapblock.cpp
In file included from jni/../jni/src/client/content_mapblock.cpp:23:
In file included from jni/../jni/src/client/mapblock_mesh.h:22:
In file included from jni/src/irrlichttypes_extrabloated.h:28:
In file included from deps/irrlicht/include/IMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= content_mapnode.cpp
[armeabi-v7a] Compile++ thumb: minetest <= content_nodemeta.cpp
In file included from jni/../jni/src/content_nodemeta.cpp:23:
In file included from jni/src/inventory.h:22:
In file included from jni/src/itemdef.h:23:
In file included from jni/src/irrlichttypes_extrabloated.h:28:
In file included from deps/irrlicht/include/IMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= content_sao.cpp
In file included from jni/../jni/src/content_sao.cpp:20:
In file included from jni/src/content_sao.h:24:
In file included from jni/src/serverobject.h:25:
In file included from jni/src/inventorymanager.h:22:
In file included from jni/src/inventory.h:22:
In file included from jni/src/itemdef.h:23:
In file included from jni/src/irrlichttypes_extrabloated.h:28:
In file included from deps/irrlicht/include/IMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= content.cpp
[armeabi-v7a] Compile++ thumb: minetest <= mods.cpp
In file included from jni/../jni/src/content/mods.cpp:29:
In file included from jni/src/porting.h:36:
In file included from deps/irrlicht/include/irrlicht.h:60:
In file included from deps/irrlicht/include/IAnimatedMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= packages.cpp
In file included from jni/../jni/src/content/packages.cpp:23:
In file included from jni/src/porting.h:36:
In file included from deps/irrlicht/include/irrlicht.h:60:
In file included from deps/irrlicht/include/IAnimatedMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= subgames.cpp
In file included from jni/../jni/src/content/subgames.cpp:21:
In file included from jni/src/porting.h:36:
In file included from deps/irrlicht/include/irrlicht.h:60:
In file included from deps/irrlicht/include/IAnimatedMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= convert_json.cpp
In file included from jni/../jni/src/convert_json.cpp:30:
In file included from jni/src/porting.h:36:
In file included from deps/irrlicht/include/irrlicht.h:60:
In file included from deps/irrlicht/include/IAnimatedMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= craftdef.cpp
In file included from jni/../jni/src/craftdef.cpp:20:
In file included from jni/src/craftdef.h:27:
In file included from jni/src/inventory.h:22:
In file included from jni/src/itemdef.h:23:
In file included from jni/src/irrlichttypes_extrabloated.h:28:
In file included from deps/irrlicht/include/IMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= database-dummy.cpp
[armeabi-v7a] Compile++ thumb: minetest <= database-files.cpp
In file included from jni/../jni/src/database/database-files.cpp:23:
In file included from jni/src/content_sao.h:24:
In file included from jni/src/serverobject.h:25:
In file included from jni/src/inventorymanager.h:22:
In file included from jni/src/inventory.h:22:
In file included from jni/src/itemdef.h:23:
In file included from jni/src/irrlichttypes_extrabloated.h:28:
In file included from deps/irrlicht/include/IMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= database-leveldb.cpp
[armeabi-v7a] Compile++ thumb: minetest <= database-sqlite3.cpp
In file included from jni/../jni/src/database/database-sqlite3.cpp:34:
In file included from jni/src/porting.h:36:
In file included from deps/irrlicht/include/irrlicht.h:60:
In file included from deps/irrlicht/include/IAnimatedMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= database.cpp
[armeabi-v7a] Compile++ thumb: minetest <= debug.cpp
In file included from jni/../jni/src/debug.cpp:21:
In file included from jni/src/porting.h:36:
In file included from deps/irrlicht/include/irrlicht.h:60:
In file included from deps/irrlicht/include/IAnimatedMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= defaultsettings.cpp
In file included from jni/../jni/src/defaultsettings.cpp:21:
In file included from jni/src/porting.h:36:
In file included from deps/irrlicht/include/irrlicht.h:60:
In file included from deps/irrlicht/include/IAnimatedMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= dungeongen.cpp
[armeabi-v7a] Compile++ thumb: minetest <= emerge.cpp
In file included from jni/../jni/src/emerge.cpp:28:
In file included from jni/src/util/thread.h:25:
In file included from jni/src/porting.h:36:
In file included from deps/irrlicht/include/irrlicht.h:60:
In file included from deps/irrlicht/include/IAnimatedMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= environment.cpp
In file included from jni/../jni/src/environment.cpp:24:
In file included from jni/src/serverobject.h:25:
In file included from jni/src/inventorymanager.h:22:
In file included from jni/src/inventory.h:22:
In file included from jni/src/itemdef.h:23:
In file included from jni/src/irrlichttypes_extrabloated.h:28:
In file included from deps/irrlicht/include/IMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= face_position_cache.cpp
[armeabi-v7a] Compile++ thumb: minetest <= filecache.cpp
[armeabi-v7a] Compile++ thumb: minetest <= filesys.cpp
In file included from jni/../jni/src/filesys.cpp:29:
In file included from jni/src/porting.h:36:
In file included from deps/irrlicht/include/irrlicht.h:60:
In file included from deps/irrlicht/include/IAnimatedMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= fontengine.cpp
In file included from jni/../jni/src/client/fontengine.cpp:22:
In file included from jni/src/client/renderingengine.h:26:
In file included from jni/src/irrlichttypes_extrabloated.h:28:
In file included from deps/irrlicht/include/IMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
1 warning generated.
[armeabi-v7a] Compile++ thumb: minetest <= game.cpp
In file included from jni/../jni/src/client/game.cpp:24:
In file included from jni/src/client/renderingengine.h:26:
In file included from jni/src/irrlichttypes_extrabloated.h:28:
In file included from deps/irrlicht/include/IMeshSceneNode.h:8:
In file included from deps/irrlicht/include/ISceneNode.h:12:
deps/irrlicht/include/ISceneNodeAnimator.h:58:16: warning: 'OnEvent' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
                virtual bool OnEvent(const SEvent& event)
                             ^
deps/irrlicht/include/IEventReceiver.h:576:15: note: overridden virtual function is here
        virtual bool OnEvent(const SEvent& event) = 0;
                     ^
In file included from jni/../jni/src/client/game.cpp:29:
In file included from jni/src/client/inputhandler.h:29:
jni/src/gui/touchscreengui.h:30:10: fatal error: 'game.h' file not found
#include "game.h"
         ^~~~~~~~
1 warning and 1 error generated.
make[2]: *** [obj/local/armeabi-v7a/objs-debug/minetest/jni/src/client/game.o] Error 1
make[2]: Leaving directory `/home/juozas/minetest/build/android'
Makefile:776: recipe for target 'apk' failed
make[1]: *** [apk] Error 1
make[1]: Leaving directory '/home/juozas/minetest/build/android'
Makefile:160: recipe for target 'debug' failed
make: *** [debug] Error 2
```

</details>

**Update:** A pr that fixes this and other possible subsequent build issues related to 5f1cd555cd9d1c64426e173b30b5b792d211c835 is now posted (#7934).

##### Steps to reproduce
<!-- For bug reports or build issues, explain how the problem happened -->

1. Start build environment such a virtual machine where all android build dependencies were already installed, e.g. one like set up according to solving of #7797 
2. Make sure previous builds were ok and were installing properly
3. Get the latest version of master (a7025c72730adcd41648613cdad93f4e6507c3b4)
4. Cd to `<minetest dir>/build/android`
5. Run make clean & make debug
6. Wait for error

--- a/src/gui/touchscreengui.cpp
+++ b/src/gui/touchscreengui.cpp
@@ -22,12 +22,12 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "irrlichttypes.h"
 #include "irr_v2d.h"
 #include "log.h"
-#include "keycode.h"
+#include "client/keycode.h"
 #include "settings.h"
 #include "gettime.h"
 #include "util/numeric.h"
 #include "porting.h"
-#include "guiscalingfilter.h"
+#include "client/guiscalingfilter.h"
 
 #include <iostream>
 #include <algorithm>

--- a/src/gui/touchscreengui.h
+++ b/src/gui/touchscreengui.h
@@ -27,7 +27,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include <vector>
 
 #include "client/tile.h"
-#include "game.h"
+#include "client/game.h"
 
 using namespace irr;
 using namespace irr::core;

